pdfminer/converter.py

```diff
@@ -358,8 +358,11 @@ def put_text(self, text, fontname, fontsize):
         if font != self._font:
             if self._font is not None:
                 self.write('</span>')
+            # Remove subset tag from fontname, see PDF Reference 5.5.3
+            fontname_without_subset_tag = fontname.split('+')[-1]
             self.write('<span style="font-family: %s; font-size:%dpx">' %
-                       (enc(fontname), fontsize * self.scale * self.fontscale))
+                       (fontname_without_subset_tag,
```

Comment:
Is the `enc` function not required for `fontname_without_subset_tag`?

Reply:
Hhm, maybe you are right. I removed it because it creates a bytes object and is written e.g. as: 

```html
<span style="font-family: b'Georgia'">
```

I will look into this.